api/onboarding.ts

```
@@ -0,0 +1,55 @@
+import process from "node:process";
+import { createClient } from "@supabase/supabase-js";
+import { authenticateSupabaseRequest } from "../src/server/private-cloud/auth.js";
+import {
+  ApiError,
+  handleJsonPost,
+  requireExactObject,
+} from "../src/server/private-cloud/http.js";
```

ESLint

[error] Replace `⏎··ApiError,⏎··handleJsonPost,⏎··requireExactObject,⏎` with `·ApiError,·handleJsonPost,·requireExactObject·`

(prettier/prettier)